src/components/AddBook/addbook.js

```diff
@@ -23,24 +23,28 @@ const AddBook = () => {
 
```

Comment:
![image](https://user-images.githubusercontent.com/93039359/230584815-94cdc3d2-2d91-45f2-a70f-37c735be39ba.png)

**[OPTIONAL]**

- [ ] It would be great if you make your Header not overflow from your application because if you see the navbar going out of the screen from the left side 😨 Kindly fix this issue to make your project look professional 👍 My apologies the previous reviewer didn't mention it but take this recommendation seriously 👍

Reply:
@HAMMAS-SALEEM  Thank you for the great and accurate observation,I will implement them right away✍